workspaces/src/worker/mod.rs

```diff
@@ -53,6 +53,11 @@ pub async fn mainnet_archival() -> anyhow::Result<Worker<Mainnet>> {
     Ok(Worker::new(Mainnet::archival().await?))
 }
 
+/// Connect to the betanet network, and grab a [`Worker`] that can interact with it.
+pub async fn betanet() -> anyhow::Result<Worker<Betanet>> {
```

Comment:
note that you didn't add the `with_betanet` option to this, but probably fine without

Reply:
added it since might as well consistent with it

Reply:
added it since might as well consistent with it